test/Server.test.js

```diff
@@ -74,6 +75,51 @@ describe('Server', () => {
     });
   });
 
+  describe('Testing callback functions on calling invalidate without callback', () => {
+    it('should be `noop` (the default callback function)', () => {
```

Comment:
Please use done because Server.test.js has already used `done` and can reduce code length.
After all, `resolve` is called in `server.close` in the callback, so you don't have to make a Promise.